Cargo.lock

```diff
@@ -619,6 +620,23 @@ dependencies = [
 ]
 
 [[package]]
+name = "ethcore-secstore"
+version = "1.0.0"
+dependencies = [
+ "ethcore-devtools 1.6.0",
+ "ethcore-ipc 1.6.0",
+ "ethcore-ipc-codegen 1.6.0",
+ "ethcore-ipc-nano 1.6.0",
+ "ethcore-util 1.6.0",
+ "ethcrypto 0.1.0",
+ "ethkey 0.2.0",
+ "hyper 0.9.14 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
Not actually :( By 'our async version' do you mean https://github.com/ethcore/hyper ? I'll take a look into it. tokio-minihttp sounds very good, as it has 'mini' in its name :) I'll check it out - thanks